### PR TITLE
fix: Name of team plan

### PR DESCRIPTION
**Pull request by IchordeDionysos**
I did leave the enum value for the `Subscription` as it was...
Not sure what the current status is here:
https://github.com/dnsimple/dnsimple-developer/blob/3124e948fc685f25c91a41931fb300906a5be824/content/v2/openapi.yml#L5050

I actually also wasn't sure what to do with those `dnsimple-professional` plan ids.

### CODE REVIEW

**Comment by sbastn** [comment]
Thank you, @IchordeDionysos, for submitting these changes. Keeping the Enum values as they are makes sense, especially since we still have customers subscribed to the Professional plan (now considered legacy).

@san983, everything looks good on my end. I do have a question about the fixtures—are they purely for illustrative purposes, or do we have tests or code that depend on the specific values defined in them?

**Review by sbastn — APPROVED** [review]
Approving, pending @san983's review

**Review by san983 — APPROVED** [review]

**Comment by san983** [comment]
> @san983, everything looks good on my end. I do have a question about the fixtures—are they purely for illustrative purposes, or do we have tests or code that depend on the specific values defined in them?

LGTM too. AFAIS this is just illustrative. But you'd have to double check with anyone from the app team.

**Comment by sbastn** [comment]
@jacegu, I tagged you for a review to make sure the changes to the fixtures are OK and won't cause any issues if there’s code or tests relying on them (which I don't think we have, but I'm not 100% sure). Thanks!

File: content/v2/certificates.markdown [hunk 1]
```diff
@@ -198,14 +198,14 @@ By default, a certificate doesn't have alternate names.
...
-Alternate names require a subscription to a Professional or Enterprise plan.
+Alternate names require a subscription to a Team or Enterprise plan.
 
 ### Wildcard certificates
 
 To request a wildcard certificate that's valid for an unlimited number of names that belong to a single subdomain level, use `*` (e.g. `*.example.com`).
 
 <info>
-Let's Encrypt wildcard certificates is a feature that is only available to the Professional or Enterprise plans.
+Let's Encrypt wildcard certificates is a feature that is only available to the Team or Enterprise plans.
```
Comment (jacegu) [inline content/v2/certificates.markdown:208]:
```suggestion
Let's Encrypt wildcard certificates is a feature that is only available to the Teams or Enterprise plans.
```

File: content/v2/certificates.markdown [hunk 2]
```diff
@@ -198,14 +198,14 @@ By default, a certificate doesn't have alternate names.
 
 You can purchase a single certificate for both `https://docs.example.com` and `https://status.example.com`, alongside  `https://example.com`.
 
-Alternate names require a subscription to a Professional or Enterprise plan.
+Alternate names require a subscription to a Team or Enterprise plan.
```
Comment (jacegu) [inline content/v2/certificates.markdown:201]:
```suggestion
Alternate names require a subscription to a Teams or Enterprise plan.
```

File: content/v2/certificates.markdown [hunk 3]
```diff
@@ -188,7 +188,7 @@ The domain must be [delegated](https://support.dnsimple.com/articles/pointing-do
 The default certificate name is `www` and covers both the root domain (e.g. `example.com`) and the `www` subdomain (e.g. `www.example.com`).
 
 You can choose a **custom name** (like `api`), which is valid only for `https://api.example.com`.
-Custom names require a subscription to a Professional or Enterprise plan.
+Custom names require a subscription to a Team or Enterprise plan.
```
Comment (jacegu) [inline content/v2/certificates.markdown:191]:
```suggestion
Custom names require a subscription to a Teams or Enterprise plan.
```

File: fixtures/v2/api/accounts/success-user.http [hunk 4]
```diff
@@ -17,4 +17,4 @@ X-Permitted-Cross-Domain-Policies: none
 X-XSS-Protection: 1; mode=block
 Strict-Transport-Security: max-age=31536000
 
-{"data":[{"id":123,"email":"john@example.com","plan_identifier":"dnsimple-personal","created_at":"2011-09-11T17:15:58Z","updated_at":"2016-06-03T15:02:26Z"},{"id":456,"email":"ops@company.com","plan_identifier":"dnsimple-professional","created_at":"2012-03-16T16:02:54Z","updated_at":"2016-06-14T11:23:16Z"}]}
+{"data":[{"id":123,"email":"john@example.com","plan_identifier":"dnsimple-personal","created_at":"2011-09-11T17:15:58Z","updated_at":"2016-06-03T15:02:26Z"},{"id":456,"email":"ops@company.com","plan_identifier":"dnsimple-teams","created_at":"2012-03-16T16:02:54Z","updated_at":"2016-06-14T11:23:16Z"}]}
```
Comment (jacegu) [inline fixtures/v2/api/accounts/success-user.http:20]:
```suggestion
{"data":[{"id":123,"email":"john@example.com","plan_identifier":"dnsimple-personal","created_at":"2011-09-11T17:15:58Z","updated_at":"2016-06-03T15:02:26Z"},{"id":456,"email":"ops@company.com","plan_identifier":"teams-v1-monthly","created_at":"2012-03-16T16:02:54Z","updated_at":"2016-06-14T11:23:16Z"}]}

```

File: fixtures/v2/api/listAccounts/success-user.http [hunk 5]
```diff
@@ -17,4 +17,4 @@ X-Permitted-Cross-Domain-Policies: none
 X-XSS-Protection: 1; mode=block
 Strict-Transport-Security: max-age=31536000
 
-{"data":[{"id":123,"email":"john@example.com","plan_identifier":"dnsimple-personal","created_at":"2011-09-11T17:15:58Z","updated_at":"2016-06-03T15:02:26Z"},{"id":456,"email":"ops@company.com","plan_identifier":"dnsimple-professional","created_at":"2012-03-16T16:02:54Z","updated_at":"2016-06-14T11:23:16Z"}]}
+{"data":[{"id":123,"email":"john@example.com","plan_identifier":"dnsimple-personal","created_at":"2011-09-11T17:15:58Z","updated_at":"2016-06-03T15:02:26Z"},{"id":456,"email":"ops@company.com","plan_identifier":"dnsimple-teams","created_at":"2012-03-16T16:02:54Z","updated_at":"2016-06-14T11:23:16Z"}]}
```
Comment (jacegu) [inline fixtures/v2/api/listAccounts/success-user.http:20]:
```suggestion
{"data":[{"id":123,"email":"john@example.com","plan_identifier":"dnsimple-personal","created_at":"2011-09-11T17:15:58Z","updated_at":"2016-06-03T15:02:26Z"},{"id":456,"email":"ops@company.com","plan_identifier":"teams-v1-monthly","created_at":"2012-03-16T16:02:54Z","updated_at":"2016-06-14T11:23:16Z"}]}

```

File: fixtures/v2/api/whoami/success-account.http [hunk 6]
```diff
@@ -12,4 +12,4 @@ X-Request-Id: 15a7f3a5-7ee5-4e36-ac5a-8c21c2e1fffd
 X-Runtime: 0.141588
 Strict-Transport-Security: max-age=31536000
 
-{"data":{"user":null,"account":{"id":1,"email":"example-account@example.com","plan_identifier":"dnsimple-professional","created_at":"2015-09-18T23:04:37Z","updated_at":"2016-06-09T20:03:39Z"}}}
+{"data":{"user":null,"account":{"id":1,"email":"example-account@example.com","plan_identifier":"dnsimple-teams","created_at":"2015-09-18T23:04:37Z","updated_at":"2016-06-09T20:03:39Z"}}}
```
Comment (jacegu) [inline fixtures/v2/api/whoami/success-account.http:15]:
```suggestion
{"data":{"user":null,"account":{"id":1,"email":"example-account@example.com","plan_identifier":"teams-v1-monthly","created_at":"2015-09-18T23:04:37Z","updated_at":"2016-06-09T20:03:39Z"}}}

```

File: fixtures/v2/api/whoami/success.http [hunk 7]
```diff
@@ -12,4 +12,4 @@ X-Request-Id: 15a7f3a5-7ee5-4e36-ac5a-8c21c2e1fffd
 X-Runtime: 0.141588
 Strict-Transport-Security: max-age=31536000
 
-{"data":{"user":null,"account":{"id":1,"email":"example-account@example.com","plan_identifier":"dnsimple-professional","created_at":"2015-09-18T23:04:37Z","updated_at":"2016-06-09T20:03:39Z"}}}
+{"data":{"user":null,"account":{"id":1,"email":"example-account@example.com","plan_identifier":"dnsimple-teams","created_at":"2015-09-18T23:04:37Z","updated_at":"2016-06-09T20:03:39Z"}}}
```
Comment (jacegu) [inline fixtures/v2/api/whoami/success.http:15]:
```suggestion
{"data":{"user":null,"account":{"id":1,"email":"example-account@example.com","plan_identifier":"teams-v1-monthly","created_at":"2015-09-18T23:04:37Z","updated_at":"2016-06-09T20:03:39Z"}}}

```

File: content/v2/openapi.yml [hunk 8]
```diff
@@ -3378,7 +3378,7 @@ components:
       example:
         id: 1
         email: example-account@example.com
-        plan_identifier: dnsimple-professional
+        plan_identifier: dnsimple-teams
```
Comment (jacegu) [inline content/v2/openapi.yml:3381]:
```suggestion
        plan_identifier: teams-v1-monthly
```

**Review by jacegu — COMMENTED** [review]
@IchordeDionysos thanks for submitting these changes.

As @sbastn mentioned, this is mostly informational. I have suggested a couple changes to your diff:
- Always referring to the Teams plan as _Teams_ (we never refer to it in singular).
- Using `teams-v1-monthly` as the plan identifier, as we don't really have a `dnsimple-teams` plan.

**Comment by IchordeDionysos** [comment]
@jacegu I applied your suggestions

**Review by jacegu — APPROVED** [review]

**Pull request merged**